src/DonorProfiles/resources/views/successnotice.php

```diff
@@ -1,11 +1,11 @@
 <?php
 
-$pageId = give_get_option( 'donor_profile_page' );
+$pageId = give_get_option( 'donor_dashboard_page' );
 
 $pageUrl = get_permalink( $pageId );
 
 ?>
 
 <div class="notice notice-success is-dismissible">
-	<p><?php printf( __( 'Success! Donor Profile page was created. You can <a href="%s">take a look at it here.</a>' ), $pageUrl ); ?></p>
+	<p><?php printf( __( 'Success! Donor Dashboard page was created. You can <a href="%s">take a look at it here.</a>' ), $pageUrl ); ?></p>
```

Comment:
Can we localize this string?

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5708/commits/d68a55292442e50d45fb4be16dfc7b0cf25baf26